src/Task/Git/CommitMessage.php

```diff
@@ -135,8 +144,7 @@ public function run(ContextInterface $context): TaskResultInterface
             );
         }
 
-
-        if ($this->enforceTypeScopeConventions()) {
+        if (!$this->isMergeCommit($commitMessage) && $this->enforceTypeScopeConventions()) {
```

Comment:
instead of checking this twice, you could as well remember it in a var like `$isMergeCommit`